chain/beacon/node.go

```diff
@@ -8,6 +8,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/drand/drand/metrics"
```

Comment:
This should have failed linting

Reply:
hmm it didn't - have moved it to where it should be anyway